packages/lesswrong/components/events/EventsHome.tsx

```diff
@@ -4,18 +4,18 @@ import { useUserLocation } from '../../lib/collections/users/helpers';
 import { useCurrentUser } from '../common/withUser';
 import { createStyles } from '@material-ui/core/styles';
 import * as _ from 'underscore';
-import NotificationsNoneIcon from '@material-ui/icons/NotificationsNone';
+import AddAlertIcon from '@material-ui/icons/AddAlert';
```

Comment:
I would have preferred to use the `NotificationAdd` icon but unfortunately we don't have the latest MUI icons: https://mui.com/components/material-icons/?query=notification